hits.tex

```diff
@@ -335,8 +335,12 @@ \section{Induction principles and dependent paths}
 \end{lem}
 \begin{proof}
   We have a canonical function $f:(\Sn^1 \to A) \to \sm{x:A} (x=x)$ defined by $f(g) \defeq (g(\base),\ap g \lloop)$.
-  The induction principle shows that the fibers of $f$ are inhabited, while the uniqueness principle shows that they are mere propositions.
-  Hence they are contractible, so $f$ is an equivalence.
+  On the other direction, we have $g:\sm{x:A} (x=x) \to (\Sn^1 \to A)$ defined by taking a pair $(b,\ell)$ to the function $\Sn^1 \to A$ given by the recursion principle of the circle.
+
+  Now, by the computation rule of the recursion principle, $f \circ g \htpy \idfunc$;
+  while $g \circ f \htpy \idfunc$, by the uniqueness principle,
+  since \((g \circ f)(\lloop) =^{\lam{x} x=_Ax}_{\refl} \lloop\), again, by the computation rule of the recursion principle of the circle.
```

Comment:
```suggestion
  since \((g \circ f)(\lloop) =^{\lam{x} x=_Ax}_{\refl{\base}} \lloop\), again, by the computation rule of the recursion principle of the circle.
```
`\refl` takes an argument.